packages/test/src/test-integration-workflows.ts

```diff
@@ -282,3 +283,34 @@ test('Activity initialInterval is not getting rounded', async (t) => {
   const retryPolicy = activityTaskScheduledEvents?.activityTaskScheduledEventAttributes?.retryPolicy;
   t.is(tsToMs(retryPolicy?.initialInterval), 50);
 });
+
+test('Start of workflow is delayed', async (t) => {
+  const { startWorkflow } = helpers(t);
+  // This workflow never runs
+  const handle = await startWorkflow(runTestActivity, {
+    startDelay: '5678s',
+  });
+  const { events } = await handle.fetchHistory();
+  const workflowExecutionStartedEvent = events?.find((ev) => ev.workflowExecutionStartedEventAttributes);
+  console.log(workflowExecutionStartedEvent);
```

Comment:
I think you meant to delete the `console.log` calls here and below.

Reply:
Oops... Thanks